src/plays/react-multistep-form/react-multistep-form.css

```diff
@@ -0,0 +1,62 @@
+.multistep-button {
+  border-radius: 8px;
+  border: 1px solid;
+  padding: 0.2em 0.5em;
+  font-size: 1em;
+  font-weight: 500;
+  font-family: inherit;
+  cursor: pointer;
+  transition: border-color 0.25s;
+}
+.multistep-button:hover {
+  border-color: #646cff;
+}
+.multistep-button:focus,
+.multistep-button:focus-visible {
+  outline: 4px auto -webkit-focus-ring-color;
+}
+
+.form-input-text {
+  width: 100%;
+  padding: 10px 10px;
+  margin: 8px 0;
+  display: inline-block;
+  border-radius: 4px;
+  box-sizing: border-box;
+  border: 1px solid black;
+  font-size: 0.75rem;
+}
+
+.container-flexbox {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+}
+
+.multistep-container {
+  border: 1px solid black;
+  position: relative;
+  background: white;
+  padding: 1rem;
+  border-radius: 0.5rem;
+  font-family: Arial;
+  max-width: 500px;
+  margin: 0 auto;
+  margin-top: 10%;
+}
+.steps {
```

Comment:
You can still make these names unique like `.multistep-play-steps` or add a parent class like `.multistep-play` and use that as prefix to all like `.multistep-play .steps`.

Reply:
Sure. Thank you.